cgd_tx_eff/src/edu/ohsu/compbio/txeff/tx_eff_control.py

```diff
@@ -74,25 +78,28 @@ def _main():
     
     args = _parse_args()
 
-    # Use tx_eff_annovar to read annovar records 
-    annovar_records = tx_eff_annovar.get_annovar_records(args.annovar_variant_function.name, args.annovar_exonic_variant_function.name)
+    # Use tx_eff_annovar to read annovar records
+    
+    annovar_records = TxEffAnnovar().get_annovar_records(args.annovar_variant_function.name, args.annovar_exonic_variant_function.name)
 
     # Load the reference genome with pysam.
     pysam_file = PysamTxEff(args.reference_fasta)
 
+    txEffHgvs = TxEffHgvs()
```

Comment:
Variables in this file seem to predominantly use snake case, not camel case. I'm not sure which is correct, or if there is even a convention.

Reply:
I've been naming classes wrong. The class name ``TxEffHgvs`` is correct, but the instance should be tx_eff_hgvs

Reply:
I just went through and fixed several class instance variables to snake case.